es-space-mgmt/main/default/aura/reservationHelperAura/reservationHelperAuraHelper.js

```diff
@@ -9,23 +9,28 @@
             { name: 'varSobjectType', type: 'String', value: sobj },
             { name: 'varState', type: 'String', value: state }
         ];
+
         if (flow) {
             flow.startFlow('createReservation', inputVariables);
         }
     },
     refreshTab: function (component) {
         var lwcHelper = component.find('reservHelperLWC');
         var sobj = component.get('v.sobjectType');
+
         lwcHelper.handleFlowExit({ detail: sobj });
         var navItemAPI = component.find('navigationItemAPI');
+
         navItemAPI
             .refreshNavigationItem()
             .then(function (response) {
```

Comment:
I suggest that we drop that logging block since we're not doing anything with it.
If we want to keep it, perhaps we could show a toast?